checker/src/main/java/org/checkerframework/checker/calledmethods/CalledMethodsTransfer.java

```diff
@@ -50,6 +52,9 @@ public class CalledMethodsTransfer extends AccumulationTransfer {
    */
   private final ExecutableElement calledMethodsValueElement;
 
+  /** True if -AenableWPIForRLC was passed on the command line. */
```

Comment:
Why does this add a new command-line argument rather than re-using the existing `-Ainfer` command-line argument?  Reusing the existing argument would be simpler for users and less error-prone, I feel.

If `-Ainfer` is passed and the RLC is running, then this code should be used.

Reply:
If users include the -Ainfer flag, this inference mechanism will be used as the default for RLC. However, in the future, when WPI and the RLC work better together, this command-line option can be useful for executing RLC inference using WPI.

Reply:
I don't quite understand the difference (which might not exist yet -- maybe it will only exist in the future).  Can you document the difference among the four possibilities (including or excluding each of the two command-line options)?